sdk/samples/iot/pnp/pnp_protocol.c

```diff
@@ -1,9 +1,9 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
-#include "iot_sample_common.h"
+#include "../iot_sample_common.h"
```

Comment:
I'm curious if we can update cmake to recognize a path like "samples/iot/iot_sample_common.h".  While "../" works, I don't normally see that used in an include.

Reply:
good point. updated feel free to check